packages/clerk-js/src/ui/components/AccountlessPrompt/index.tsx

```diff
@@ -1,169 +1,397 @@
-import type { PointerEventHandler } from 'react';
-import { useCallback, useEffect, useRef } from 'react';
+// eslint-disable-next-line no-restricted-imports
+import { css } from '@emotion/react';
+import { useState } from 'react';
 
-import type { LocalizationKey } from '../../customizables';
-import { Col, descriptors, Flex, Link, Text } from '../../customizables';
+import { descriptors, Flex, Link } from '../../customizables';
 import { Portal } from '../../elements/Portal';
-import { InternalThemeProvider, mqu } from '../../styledSystem';
+import { InternalThemeProvider } from '../../styledSystem';
+import { ClerkLogoIcon } from './ClerkLogoIcon';
+import { KeySlashIcon } from './KeySlashIcon';
 
 type AccountlessPromptProps = {
   url?: string;
 };
 
-type FabContentProps = { title?: LocalizationKey | string; signOutText: LocalizationKey | string; url: string };
+export const _AccountlessPrompt = (_props: AccountlessPromptProps) => {
+  const [isExpanded, setIsExpanded] = useState(true);
 
-const FabContent = ({ title, signOutText, url }: FabContentProps) => {
-  return (
-    <Col
-      sx={t => ({
-        width: '100%',
-        paddingLeft: t.sizes.$4,
-        paddingRight: t.sizes.$6,
-        whiteSpace: 'nowrap',
-      })}
-    >
-      <Text
-        colorScheme='secondary'
-        elementDescriptor={descriptors.impersonationFabTitle}
-        variant='buttonLarge'
-        truncate
-        localizationKey={title}
-      />
-      <Link
-        variant='buttonLarge'
-        elementDescriptor={descriptors.impersonationFabActionLink}
-        sx={t => ({
-          alignSelf: 'flex-start',
-          color: t.colors.$primary500,
-          ':hover': {
-            cursor: 'pointer',
-          },
-        })}
-        localizationKey={signOutText}
-        onClick={
-          () => (window.location.href = url)
-          // clerk-js has been loaded at this point so we can safely access session
-          // handleSignOutSessionClicked(session!)
-        }
-      />
-    </Col>
-  );
-};
+  const bottomProperty = '--cl-keyless-bottom';
+  const rightProperty = '--cl-keyless-right';
+  const defaultBottom = 50;
+  const defaultRight = 50;
```

Comment:
Whats the reason for the variables usage here? I am not seeing these update or conditionally rendered at all.